generators/typescript/sdk/generator/src/test-generator/JestTestGenerator.ts

```diff
@@ -398,3 +402,30 @@ describe("test", () => {
           `;
     }
 }
+
+function getExampleTypeReferenceForResponse(exampleResponse: IR.ExampleResponse): IR.ExampleTypeReference | undefined {
+    switch (exampleResponse.type) {
+        case "ok":
+            return getExampleTypeReferenceForSuccessResponse(exampleResponse.value);
+        case "error":
+            return exampleResponse.body;
+        default:
+            assertNever(exampleResponse);
+    }
+}
+
+// TODO: Update this to handle multiple responses in the stream and sse cases.
```

Comment:
@dsinghvi FYI here - we'll need to refactor the jest test generator to handle these cases, so I figured this was the best fallback for now.